types/alpm-definition.json

```diff
@@ -36,11 +41,24 @@
       "requirement": "optional",
       "native_name": "arch",
       "description": "The arch is the qualifiers key for a package architecture."
+    },
+    {
+      "key": "distro",
+      "requirement": "optional",
+      "description": "The distribution name when using multiple distributions."
+    },
+    {
+      "key": "repository_url",
```

Comment:
This is a standard qualifier so it not really needed here IMHO, but it does not hurt either... @johnmhoran what do you think?